kythe/cxx/verifier/verifier.h

```diff
@@ -336,6 +339,9 @@ class Verifier {
   /// If true, show anchor locations in graph dumps (instead of @).
   bool show_anchors_ = false;
 
+  /// If true, show unlabeled nodes in graph dumps.
+  bool show_unlabled_ = true;
```

Comment:
```suggestion
  bool show_unlabeled_ = true;
```

Reply:
That's what I get for coding on a weekend, thanks!